src/ewatercycle/forcing/_wflow.py

```diff
@@ -71,7 +70,7 @@ def generate(  # type: ignore
         script["dem_file"] = dem_file
 
         if extract_region is None:
-            extract_region = get_extents(shape)
+            extract_region = get_extents(shape, pad=3)
```

Comment:
Why choose `3` as the value of pad? Are `1` and `2` not enough? Why not choose for an even larger value?

Reply:
Good question. I know the resolution of ERA-Interim is 0.75 degrees. I'm guessing (but am not sure) that you might need at least two grid cells for conservative regridding, so that would be 1.5 degrees. Of course, we're not sure that the edge of the shapefile exactly coincides with a grid cell center coordinate, so a bit of extra spacing would give us some leeway. I tried with 3 and 5 for the Savannah catchment. Both worked fine, so I chose the smaller one.